README.md

```diff
@@ -25,7 +25,6 @@ module "ecr" {
   source = "hazelops/ecr/aws"
```

Comment:
Let's change `Minimal` to `Miminal & Unrestricted` and `Full` to `Full & Restricted by IAM arns` Or something like that

Reply:
Fixed